scripts/revcheck.php

```diff
@@ -190,6 +190,39 @@ function populateFileTreeRecurse( $lang , $path , & $output )
         }
         if ( $entry->isFile() )
         {
+            $ignoredFileNames = [
+                'README.md',      
+                'translation.xml',      
+                'readme.first',      
+                'license.xml',      
+                'extensions.xml',
+                'versions.xml',
+                'book.developer.xml',
+                'contributors.ent',
+                'contributors.xml',
+                'README',
+                'DO_NOT_TRANSLATE',
+                'rsusi.txt',
+                'missing-ids.xml',
+            ];
+            
+            $ignoredDirectories = [
+                'chmonly',
+                'internals',
+                'internals2',
+            ];
+            
+            if(
+                in_array($trimPath, $ignoredDirectories, true) 
+                || in_array($filename, $ignoredFileNames, true)
+                || (strpos($filename, 'entities.') === 0)
```

Comment:
Are we not running PHP 8 yet? 
Also, the last two `in_array` are missing the strict flag

Reply:
Thanks! Added strict flag, but I'm not sure about PHP 8.

Reply:
This script is supposed to be run locally, AFAIK. Raising requirements to PHP 8.0 might be a bit early; at least that should be discussed on the docs ML.